### ISSUE TITLE
Migrate database to use Django ORM.

### ISSUE THREAD

**Issue by nicomiguelino** [Screenly/Anthias#1913]
#### Overview

- Depends on #558.
- Database manipulation is currently handled using the `sqlite3` Python library.
- The database should be migrated to use the Django ORM.

**Comment by nicomiguelino** [comment]
Changes were already merged into the `experimental` branch. A separate pull request will be created for merging the `experimental` branch with the `master` branch.
